src/main/java/uk/gov/hmcts/reform/et/syaapi/service/SendNotificationService.java

```diff
@@ -156,4 +171,14 @@ private void sendAddResponseSendNotificationEmails(CaseData caseData,
 
     }
 
+    private void setResponsesAsRespondedTo(List<GenericTypeItem<RespondNotificationType>> responses) {
+        if (CollectionUtils.isEmpty(responses)) {
```

Comment:
Can responses be null?

Reply:
A test to cover when responses are empty would do no harm

Reply:
isEmpty is null safe, it will check for null too, but I'll add a test too, good practice